Sources/MongoSwift/MongoClient.swift

```diff
@@ -304,6 +304,107 @@ public class MongoClient {
         return MongoDatabase(name: name, client: self, options: options)
     }
 
+     /**
+      * Starts a `ChangeStream` on a `MongoClient`. Allows the client to observe all changes in a cluster - excluding
+      * system collections i.g. "config", "local", and "admin" databases. By default, the type `Document` is
```

Comment:
I think we could omit the "by default, ..." sentence. it implies that this method has some way to override that behavior, whereas really they need to use another method.